addon/components/md-modal.js

```diff
@@ -26,19 +29,9 @@ export default Component.extend(UsesSettings, {
     return names.join(' ');
   }),
 
-  didInsertElement() {
-    this._super(...arguments);
-    this.becomeKeyResponder();
-  },
-
-  willDestroyElement() {
-    this._super(...arguments);
-    this.resignKeyResponder();
-  },
-
-  cancel() {
-    this.sendAction('close');
```

Comment:
People may expect this action to be fired. Your current code omits this action.

Please add a test to prove that this functionality works as intended